.github/workflows/user_docs.yml

```
@@ -0,0 +1,31 @@
+name: Update User Docs
+
+on:
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: windows-latest
+
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
```

Very cosmetic, but we're already at version 4 of this action:
```suggestion
      uses: actions/checkout@v4
```